src/crypto/crypto_util.cc

```diff
@@ -201,6 +203,27 @@ void TestFipsCrypto(const v8::FunctionCallbackInfo<v8::Value>& args) {
   args.GetReturnValue().Set(ncrypto::testFipsEnabled() ? 1 : 0);
 }
 
+void GetOpenSSLSecLevelCrypto(const FunctionCallbackInfo<Value>& args) {
+  // for BoringSSL assume the same as the default
+  int sec_level = 1;
```

Comment:
Maybe this should be `OPENSSL_TLS_SECURITY_LEVEL`?
https://github.com/nodejs/node/blob/7bc2946293757389468f1fa09714860f8e1147b7/deps/openssl/openssl/include/openssl/tls1.h#L29-L32

While `1` is the default for OpenSSL 3.0 and 3.1, [OpenSSL 3.2 changed the default to `2`](https://docs.openssl.org/3.2/man3/SSL_CTX_set_security_level/#notes).